src/Controls/src/Xaml/Hosting/AppHostBuilderExtensions.cs

```diff
@@ -62,101 +62,112 @@ public static partial class AppHostBuilderExtensions
+			handlersCollection.AddHandler<Switch>(_ => new SwitchHandler());
+			handlersCollection.AddHandler<TimePicker>(_ => new TimePickerHandler());
+			handlersCollection.AddHandler<Page>(_ => new PageHandler());
+			handlersCollection.AddHandler<WebView>(_ => new WebViewHandler());
+			handlersCollection.AddHandler<Border>(_ => new BorderHandler());
+			handlersCollection.AddHandler<IContentView>(_ => new ContentViewHandler());
+			handlersCollection.AddHandler<Shapes.Ellipse>(_ => new ShapeViewHandler());
+			handlersCollection.AddHandler<Shapes.Line>(_ => new LineHandler());
+			handlersCollection.AddHandler<Shapes.Path>(_ => new PathHandler());
+			handlersCollection.AddHandler<Shapes.Polygon>(_ => new PolygonHandler());
+			handlersCollection.AddHandler<Shapes.Polyline>(_ => new PolylineHandler());
+			handlersCollection.AddHandler<Shapes.Rectangle>(_ => new RectangleHandler());
+			handlersCollection.AddHandler<Shapes.RoundRectangle>(_ => new RoundRectangleHandler());
+			handlersCollection.AddHandler<Window>(_ => new WindowHandler());
+			handlersCollection.AddHandler<ImageButton>(_ => new ImageButtonHandler());
+			handlersCollection.AddHandler<IndicatorView>(_ => new IndicatorViewHandler());
+			handlersCollection.AddHandler<RadioButton>(_ => new RadioButtonHandler());
+			handlersCollection.AddHandler<RefreshView>(_ => new RefreshViewHandler());
+			handlersCollection.AddHandler<SwipeItem>(_ => new SwipeItemMenuItemHandler());
+			handlersCollection.AddHandler<SwipeView>(_ => new SwipeViewHandler());
 
 #pragma warning disable CA1416 //  'MenuBarHandler', MenuFlyoutSubItemHandler, MenuFlyoutSubItemHandler, MenuBarItemHandler is only supported on: 'ios' 13.0 and later
-			handlersCollection.AddHandler<MenuBar, MenuBarHandler>();
-			handlersCollection.AddHandler<MenuFlyoutSubItem, MenuFlyoutSubItemHandler>();
-			handlersCollection.AddHandler<MenuFlyoutSeparator, MenuFlyoutSeparatorHandler>();
-			handlersCollection.AddHandler<MenuFlyoutItem, MenuFlyoutItemHandler>();
-			handlersCollection.AddHandler<MenuBarItem, MenuBarItemHandler>();
+			handlersCollection.AddHandler<MenuBar>(_ => new MenuBarHandler());
+			handlersCollection.AddHandler<MenuFlyoutSubItem>(_ => new MenuFlyoutSubItemHandler());
+			handlersCollection.AddHandler<MenuFlyoutSeparator>(_ => new MenuFlyoutSeparatorHandler());
+			handlersCollection.AddHandler<MenuFlyoutItem>(_ => new MenuFlyoutItemHandler());
+			handlersCollection.AddHandler<MenuBarItem>(_ => new MenuBarItemHandler());
 #pragma warning restore CA1416
 
 #if WINDOWS || ANDROID || IOS || MACCATALYST || TIZEN
-			handlersCollection.AddHandler(typeof(ListView), typeof(Handlers.Compatibility.ListViewRenderer));
+			handlersCollection.AddHandler<ListView>(svc => new Handlers.Compatibility.ListViewRenderer(
+	#if ANDROID
+				svc.GetRequiredService<Android.Content.Context>()
+	#endif
+			));
 #if !TIZEN
-			handlersCollection.AddHandler(typeof(Cell), typeof(Handlers.Compatibility.CellRenderer));
```

Comment:
I don't see `Cell` anymore in the list. Is it not needed?

Reply:
Thanks! I forgot to mention this in the description or ask about it.

If I do:
```csharp
handlersCollection.AddHandler<Cell>(_ => new Handlers.Compatibility.CellRenderer());
```
I get the error on Windows:
```
src\Controls\src\Xaml\Hosting\AppHostBuilderExtensions.cs(121,45): error CS0144: Cannot create an instance of the abstract type or interface 'CellRenderer'
```
So, I guess it might not be needed? It doesn't seem like M.E.DI could even create `CellRenderer` on Windows as it is abstract? @PureWeen @mattleibow?

I put the line back with `#if !WINDOWS` around it for now.

Reply:
Shrug Emoji. But it would be good to have @PureWeen or @mattleibow let us know if it is safe to delete.

Reply:
I think we can remove the Cell registration as the cells do not support being inflated from the base renderer. Unless there was some exception in the list views?

Reply:
What's the view type being passed in that causes

> src\Controls\src\Xaml\Hosting\AppHostBuilderExtensions.cs(121,45): error CS0144: Cannot create an instance of the abstract type or interface 'CellRenderer'

Reply:
It is `Cell`:
```csharp
handlersCollection.AddHandler<Cell>(_ => new Handlers.Compatibility.CellRenderer());
```
Seems like it is `abstract` on Windows and not the other platforms:

https://github.com/dotnet/maui/blob/3f417755ea47b82b7daec807e723ea2bf6fa597d/src/Controls/src/Core/Compatibility/Handlers/ListView/Windows/TextCellRenderer.cs#L10

@mattleibow thinks I can just remove this line. So, going to revert 1 commit here.

Reply:
Right, but why is something of type "Cell" even making it this far? 

This type of exception feels like something that's a subclass of Cell that should have been handled by a different handler but those weren't registered correctly.

Reply:
Aw, ok, so yea, in that case removing it is fine :-)